### PR TITLE
Remove aws-provisioner support from UI

### LINKED ISSUE #1576: Remove aws-provisioner support from UI
**Reserved** for [outreachy](https://www.outreachy.org/) candidates. In order for this to be assigned to you, make sure to complete the first step of the application process outlined in the [project description](https://www.outreachy.org/december-2019-to-march-2020-internship-round/communities/mozilla/#build-a-component-that-can-display-a-json-schema).

---

The "AWS Provisioner" component will no longer have anything to talk to as of November 9.  Let's remove it.  This involves:
 * removing from the navigation menu
 * removing the view and any associated components
 * removing the aws-provisioner-related GraphQL queries from the `web-server` service

### PR DESCRIPTION
**Pull request by kellim**
Removed AWS Provisioner support from the UI including removing it from the navigation menu, removing associated components, and removing aws-provisioner related GraphQL queries from the web-server service. 

Used PR #929 to help me find everything that needed to be removed, however only removed /aws-provisioner related items. 

Closes #1576

### CODE REVIEW

File: services/web-server/src/clients.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 const {
   Auth,
-  AwsProvisioner,
   EC2Manager,
```
Comment (djmitche) [inline services/web-server/src/clients.js]: This, too, is no longer used (here or in `src/loaders/workerTypes.js`)

File: changelog/1576.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+level: major
```
Comment (djmitche) [inline changelog/1576.md]: Thanks for writing a changelog!

This patch seems like a big deal, but it's actually removing something that doesn't work, so it's not strictly a breaking change in the sense of semantic versioning.  It's still a pretty big deal, though :)

```suggestion
level: minor
```

File: changelog/1576.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+reference: issue 1576
+---
+
+AWS Provisioner support has been removed from the UI and it is no longer a navigation menu item.
```
Comment (djmitche) [inline changelog/1576.md]:
```suggestion
AWS Provisioner support has been removed from the UI and it is no longer a navigation menu item.
This service has not been a part of the Taskcluster deployment for some time.
```

File: ui/src/components/WorkerTypesTable/index.jsx [hunk 4]
```diff
@@ -85,18 +73,10 @@ export default class WorkerTypesTable extends Component {
...
-      sortDirection
-    ) => {
+    (workerTypesConnection, sortBy, sortDirection) => {
       const sortByProperty = camelCase(sortBy);
       // Normalize worker types for aws-provisioner-v1
-      const workerTypes = normalizeWorkerTypes(
-        workerTypesConnection,
-        awsProvisionerWorkerTypeSummaries
-      );
+      const workerTypes = normalizeWorkerTypes(workerTypesConnection);
```
Comment (helfi92) [inline ui/src/components/WorkerTypesTable/index.jsx]: We no longer need to `normalizeWorkerTypes`. We can just use `workerTypesConnection` directly.

File: services/web-server/src/clients.js (hunk 1 above)
Comment (kellim) [reply services/web-server/src/clients.js]: Thanks! I updated this in both places.

File: changelog/1576.md (hunk 3 above)
Comment (kellim) [reply changelog/1576.md]: Oops, I committed these changes twice. It's cool GitHub lets you commit suggestions with a click of a button, but then I wasn't quite sure how to update my fork.

**Comment by helfi92** [comment]
@all-contributors please add @kellim for code.

**Pull request merged**